web/templates/web/header.html

```diff
@@ -6,78 +6,92 @@
 
 <link rel="stylesheet" type="text/css" href="{% static "web/css/header.css" %}">
 
-<div class="nav">
-    <div class="burger">
-        <i class="sidebar icon"></i>
-    </div>
+<header id="header">
+    <div class="logo-centering">{# Empty div for centering logo (on mobile), specified with grid-template-columns #}</div>
+
     <div class="logo">
+        <div>{# Empty div for spacing left of logo, specified with grid-template-columns #}</div>
         <a href="{% url 'front-page' %}">
             <img class="limg large" src="{% static "web/img/logo_white.svg" %}">
```

Comment:
Do not see the need for the `limg` class here, it is only used for a selector, which actually does not need it.